curriculum/challenges/english/blocks/quiz-dynamic-programming/67f413a2abe9894c52e15c6e.md

```diff
@@ -17,439 +17,218 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 
 #### --text--
 
-Placeholder question
+What are the two essential properties that must be present in a problem for dynamic programming to be an effective solution approach?
 
 #### --distractors--
 
-Placeholder distractor 1
+Fast execution time and minimal memory usage
 
 ---
 
-Placeholder distractor 2
+Recursion capability and iterative loops
 
 ---
 
-Placeholder distractor 3
+Sequential processing and parallel computation
```

Comment:
```suggestion
Sequential processing and parallel computation.
```

Reply:
I'm not sure here the period is needed, it's not a sentence

Reply:
but if you added the period everywhere else, not adding it here would be weird, @zairahira